[+] tests

sashagnip opened the pull request.

Commit:
[+] tests

Commit:
[*]

Commit:
[*]

sashagnip committed:
[*]

sashagnip committed:
[*]

lastperson reviewed (approved):
Proceed with adding own contract and tests.

sashagnip committed:
[+] my contract with tests

sashagnip commented:
Please check out tests for my own contract

sashagnip committed:
[*] fix

sashagnip commented:
1) added "constant", removed ".call"
2) "it only giver can decrease debt" means that only the person who gave money can decrease debt amount when person who took money returns some part of them. The test is trying to call decDebt() function from a stranger and checks if it throws. Renamed to "should fail if stranger tries to decrease debt amount"

sashagnip committed:
[+] OXO

sashagnip committed:
[*] fix
